pkg/shell/plot.js

```diff
@@ -601,6 +601,9 @@ shell.plot = function plot(element, x_range_seconds, x_stop_seconds) {
     $(element).on("plotselecting", selecting);
     $(element).on("plotselected", selected);
 
+    // for testing
+    $(element).data("flot_data", flot_data);
```

Comment:
Does this create a memory leak? Where is this cleared from the element? Or is that not necessary?